File: Modulo 4 - Memoria/mario/mario.c
#include <cs50.h>
#include <stdio.h>

void draw(int h);
int main(void)
{
    int height = get_int("height: ");
    draw(height);
}

void draw(int h)
{
    if (h == 0)
    {
        return;
    }
    draw(h - 1);
    for (int i = 0; i < h; i++)
    {
        printf("#");
    }
    printf("\n");
}

File: Modulo 6 - Python/exercicio2/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for i in range(height):
    print(" " * ((height - 1) - i), end="")
    print("#" * (i+1), end="")
    print("  ", end="")
    print("#" * (i+1))